src/decoder/symphonia.rs

```diff
@@ -77,13 +77,19 @@ impl SymphoniaDecoder {
         };
 
         // Select the first supported track
-        let track_id = probed
+        let track_id = match probed
             .format
             .tracks()
             .iter()
             .find(|t| t.codec_params.codec != CODEC_TYPE_NULL)
-            .unwrap()
-            .id;
+        {
```

Comment:
Looking good, thanks again for the investigation and fix!

One more tweak you could try (if you want to) is to replace the match with something using [`Option::ok_or`](https://doc.rust-lang.org/std/option/enum.Option.html#method.ok_or), that would look something like this:

```rust
.find(...)
.ok_or(symphonia::core::errors::Error::Unsupported(...))?
.id;
```

But if you rather leave it thats fine too! This code is better then most in rodio so we are good to merge it without any more changes :)